File: Task1/java/helloworld.java
import java.text.SimpleDateFormat;
import java.util.*;

class HelloWorld {
  public static void main(String[] args) {

    //Initialize Date and colors
    //String timeStamp = (new java.util.Date()).toLocaleString();
    SimpleDateFormat dateFormat = new SimpleDateFormat("HH:mm:ss");
    Date today = new Date();
    TimeZone pst = TimeZone.getTimeZone("America/Los_Angeles");
    dateFormat.setTimeZone(pst);
    String timeStamp = dateFormat.format(today);
    String[] colors  = {"aqua", "black", "blue", "fuchsia", "gray", "green", "lime",
        "maroon", "navy", "olive","purple", "red", "silver", "teal", "white", "yellow"};
    String textColor = "black";

    //Grab random number from 0 -15
    Random rand = new Random();
    int randomNum = rand.nextInt(16);

    String backgroundColor  = colors[randomNum];

    if(backgroundColor == "black" || backgroundColor == "navy" || backgroundColor == "blue") textColor = "white";
    String document = String.format("<!doctype html> \n"
      + "<body style = 'background-color: %s'>"
      + "<h1 style = 'color:%s'> Hello World from Java @ %s </h1>"
      + "</body> "
      +"</html>", backgroundColor , textColor ,  timeStamp);

    System.out.print("Content-Type: text/html\n\n");
    System.out.print(document);
  }
}

File: Task1/helloworld.php
#!/usr/bin/php

echo 'Content-Type: text/html\n\n'
echo '<html>
		<head><meta charset="utf-8">
    	<meta name="viewport" content="width=device-width, initial-scale=1"><title>Hello World PHP</title>'
date_default_timezone_set('America/Los_Angeles');
$date = date('m/d/Y h:i:s a', time());

echo $date;
=begin
<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
	<title>Problem 1 in PHP</title>
</head>
<?php/*
	$color = array('aqua', 'black', 'blue', 'fuchsia', 'gray', 'green', 'lime', 'maroon', 'navy', 'olive', 'purple', 'red', 'silver', 'teal', 'white',  'yellow');
	$rd = rand(0,16);

	date_default_timezone_set('America/Los_Angeles');
	$date = date('m/d/Y h:i:s a', time());
	$res = "Hello World from PHP @ ".$date;
	?>
<body style="background-color: <?php echo$color[floor($rd)];?>;">
	<?php
	echo '<h1>'.$res.'</h1>';
	?>
</body>
</html>=end


